assets/dev/js/editor/utils/input-utils.js

```diff
@@ -0,0 +1,11 @@
+export default class InputUtils {
```

Comment:
Just a thought, maybe it should be something like this?

```suggestion
export default class UserEvents {
```

OR

```suggestion
export default class InputUserInteractions {
```

WDYT?